src/Language/PureScript/Errors.hs

```diff
@@ -604,6 +606,10 @@ prettyPrintSingleError full level showWiki e = flip evalState defaultUnknownMap
       paras [ line $ "Conflicting definitions are in scope for " ++ nm ++ " from the following modules:"
             , indent $ paras $ map (line . runModuleName) ms
             ]
+    renderSimpleErrorMessage (ScopeShadowing nm ms) =
+      paras [ line $ "Ignored shadowed definitions are in scope for " ++ nm ++ " from the following modules:"
```

Comment:
Does it make sense to say "from the following _open module imports_:" or something along those lines, and then to print the list with the word `import`?